core/controllers/learner_dashboard_test.py

```diff
@@ -401,6 +401,7 @@ def test_get_suggestions_after_updating_suggestion_summary(self):
         self.assertTrue(
             messages_summary['author_picture_data_url'].startswith(
                 'data:image/png;'))
+        self.assertTrue(messages_summary.get('created_on'))
```

Comment:
@Showtim3 I'm not sure about this test, why assertTrue for this and False for other fields like `description`?

I would suggest having a test that will match the `created_on` key exists in the dict and it matches the created_on date of the feedback. Let me know if you need any kind of help. Let's resolve this comment and merge this PR.

Also, can you please share an image of the UI with the date?